website/docs/r/ess_scalinggroup_vserver_groups.markdown

```diff
@@ -0,0 +1,117 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_ess_scalinggroup_vserver_groups"
+sidebar_current: "docs-alicloud-resource-ess_scalinggroup_vserver_groups"
+description: |-
+  Provides a ESS Attachment resource to attach or remove vserver groups.
+---
+
+# alicloud\_ess\_scalinggroup\_vserver\_groups
+
+Attaches/Detaches vserver groups to a specified scaling group.
+
+-> **NOTE:** Scaling group and vserver groups must be in the same region.
+
+-> **NOTE:** The load balancer of which vserver groups belongs to must be in `active` status.
+
+-> **NOTE:** If scaling group's network type is `VPC`, the vserver groups must be in the same `VPC`.
+ 
+-> **NOTE:** A scaling group can have at most 5 vserver groups attached.
+
+-> **NOTE:** Vserver groups and the default group of loadbalancer share the same backend server quota.
+
+-> **NOTE:** Resource `alicloud_ess_scalinggroup_vserver_groups` is available in 1.53.0+.
+
+## Example Usage
+
+```
+variable "name" {
+  default = "testAccEssVserverGroupsAttachment"
+}
+
+data "alicloud_zones" "default" {
+  available_disk_category     = "cloud_efficiency"
+  available_resource_creation = "VSwitch"
+}
+
+resource "alicloud_vpc" "default" {
+  name       = "${var.name}"
+  cidr_block = "172.16.0.0/16"
+}
+
+resource "alicloud_vswitch" "default" {
+  vpc_id            = "${alicloud_vpc.default.id}"
+  cidr_block        = "172.16.0.0/24"
+  availability_zone = "${data.alicloud_zones.default.zones.0.id}"
+  name              = "${var.name}"
+}
+
+resource "alicloud_slb" "default" {
+  name = "${var.name}"
+  vswitch_id = "${alicloud_vswitch.default.id}"
+}
+
+resource "alicloud_slb_server_group" "default" {
+  load_balancer_id = "${alicloud_slb.default.id}"
+  name = "test"
+}
+	
+resource "alicloud_slb_listener" "default" {
+  count = 2
+  load_balancer_id = "${element(alicloud_slb.default.*.id, count.index)}"
+  backend_port = "22"
+  frontend_port = "22"
+  protocol = "tcp"
+  bandwidth = "10"
+  health_check_type = "tcp"
+}
+
+resource "alicloud_ess_scaling_group" "default" {
+  min_size = "2"
+  max_size = "2"
+  scaling_group_name = "${var.name}"
+  vswitch_ids = ["${alicloud_vswitch.default.id}"]
+  depends_on = ["alicloud_slb_listener.default"]
+}
+
+resource "alicloud_ess_scalinggroup_vserver_groups" "default" {
+  scaling_group_id = "${alicloud_ess_scaling_group.default.id}"
+  vserver_groups {
+  loadbalancer_id = "${alicloud_slb.default.id}"
+  vserver_attributes {
+    vserver_group_id = "${alicloud_slb_server_group.default.id}"
+    port = "100"
+    weight = "60"
+    }
+  }
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `scaling_group_id` - (Required) ID of the scaling group.
+* `vserver_groups` - (Optional) The vserver groups attached on scaling group. See [Block vserver_group](#block-vserver_group) below for details.
+
+
+## Attributes Reference
```

Comment:
I think the block `Attributes Reference` should be moved the last and it only has one attribute: `id`